photon-client/src/main/scala/com/linkedin/photon/ml/io/scopt/avro/ScoptNameAndTermFeatureBagsParametersParser.scala

```diff
@@ -74,6 +75,11 @@ object ScoptNameAndTermFeatureBagsParametersParser extends ScoptParser {
       NameAndTermFeatureBagsDriver.applicationName,
       usageText = "<name>"))
 
+    // Time zone
+    ScoptParameter[DateTimeZone, DateTimeZone](
+      NameAndTermFeatureBagsDriver.timeZone,
+      usageText = "<time zone>")
```

Comment:
For each of the parsers, I would add the following to the time zone definition:
```
usageText = ... ,
additionalDocs = Seq("For a list of valid timezone ids, see: http://joda-time.sourceforge.net/timezones.html"))
```

Reply:
Done.